DC-SLES-amd-sev

```diff
@@ -0,0 +1,19 @@
+## ---------------------------- 
+## Doc Config File for SUSE Linux Enterprise AMD Secure Encrypted Virtualization
+## (AMD-SEV)
+## ----------------------------
+##
+## Basics
+MAIN="art_amd-sev.xml"
+ROOTID="art.amd-sev"
+
+## Profiling
+PROFOS="article"
+PROFARCH="x86_64"
```

Comment:
```suggestion
ADOC_POST=yes
ADOC_TYPE=article
```
Replace the two PROF lines with the above

Reply:
I don't know what this does, but: done.

I did not use the DAPS AsciiDoctor conversion, if it is connected with that. It produced very messy DocBook that I was unable to follow, so I redid it by hand. I am quite used to this now so the process is fairly fast.

Reply:
My bad - you are using DocBook now. These attributes are for ASCIIDoc processing, just ignore this remark. Sorry.

Reply:
Lines removed again. :-)